app/src/main/AndroidManifest.xml

```diff
@@ -16,14 +16,22 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-
         <activity android:name=".SignupActivity" />
         <activity
             android:name=".MainActivity"
             android:label="@string/title_activity_newsfeed"
             android:theme="@style/AppTheme.NoActionBar" />
-        <activity android:name=".Progress" />
+
         <activity android:name=".CreatePostActivity" />
+        <activity
+            android:name=".SettingsActivity"
+            android:exported="true" />
+        <activity
+            android:name=".NotificationsActivity"
+            android:exported="true" />
```

Comment:
I'd be careful adding these as it seems that android:exported="true" allows other applications to access the activity. Is that what you want here? If so please explain why you'd want other apps to be able to access account settings, account notifications, or the ability to change passwords.